src/scripts/choices.js

```diff
@@ -117,6 +117,10 @@ class Choices {
       );
     }
 
+    this.config.searchEnabled = this._isSelectOneElement
+      ? this.config.searchEnabled
+      : true;
+
```

Comment:
I think this should be simplified to [documented logic](https://github.com/jshjohnson/Choices/blob/master/README.md#searchenabled):
```js
this.config.searchEnabled = this._isSelectMultipleElement || this.config.searchEnabled;
```